.github/workflows/s3-tests.yml

```diff
@@ -0,0 +1,314 @@
+
+      - name: Copy test-env file to .env for neofs-s3-dev-env
+        run: |
+          cp .env ${GITHUB_WORKSPACE}/neofs-s3-dev-env/.env
+        working-directory: neofs-dev-env
+
+      - name: Copy neofs-s3-authmate for neofs-s3-dev-env
+        run: |
+          cp ${GITHUB_WORKSPACE}/neofs-dev-env/vendor/neofs-s3-authmate bin/s3-authmate
+        working-directory: neofs-s3-dev-env
+
+      - name: Prepare test S3-gw test env
+        run: |
+          make prepare.s3-gw
+        working-directory: neofs-s3-dev-env
+
+      - name: Copy s3tests config to s3-tests directory
+        run: |
+          cp services/s3-gw/s3tests.conf ${GITHUB_WORKSPACE}/s3-tests/${S3_TESTS_CONFIG}
+        working-directory: neofs-s3-dev-env
+
+      - name: Prepare s3tests config
+        run: |
+          sed -i -E "s/host =.*?$/host = ${S3_GW_IP}/" ${S3_TESTS_CONFIG}
+          sed -i -E "s/port =.*?$/port = ${S3_GW_PORT}/" ${S3_TESTS_CONFIG}
+          sed -i -E "s/is_secure =.*?$/is_secure = True/" ${S3_TESTS_CONFIG}
+          sed -i -E "s/ssl_verify =.*?$/ssl_verify = False/" ${S3_TESTS_CONFIG}
+        working-directory: s3-tests
+
+      - name: Test config
+        run: |
+          cat ${S3_TESTS_CONFIG}
+        working-directory: s3-tests
+
+      - name: Setup test env
+        run: |
+          python3.8 -m venv virtualenv
+          ./bootstrap
+        working-directory: s3-tests
+
+      - name: Run Sanity tests for pull requests
+        timeout-minutes: 60
+        if: github.event_name == 'pull_request'
+        run: |
+          source virtualenv/bin/activate
+          S3TEST_CONF=${S3_TESTS_CONFIG} pytest --alluredir=${GITHUB_WORKSPACE}/allure-results -v -s s3tests_boto3/functional/test_s3.py --timeout 300 -a "sanity"
+        working-directory: s3-tests
+
+      - name: Run All tests
+        if: github.event_name != 'pull_request'
+        run: |
+          source virtualenv/bin/activate
+          S3TEST_CONF=${S3_TESTS_CONFIG} pytest --alluredir=${GITHUB_WORKSPACE}/allure-results -v -s s3tests_boto3/functional/test_s3.py --timeout 300
+        working-directory: s3-tests
+
+
+################################################################
+      - name: Generate Allure report
+        timeout-minutes: 60
+        uses: simple-elf/allure-report-action@v1.6
+        if: always()
+        id: allure-report
+        with:
+          keep_reports: 100000
+          allure_results: allure-results
+          allure_report: allure-report
+          allure_history: allure-history
+
+      - name: Enable stable neofs-cli
+        if: always()
+        run: |
+          echo "$(pwd)" >> $GITHUB_PATH
+        working-directory: neofs-node-stable
+
+      - name: Create wallet
+        if: always()
+        env:
+          TEST_RESULTS_WALLET: ${{ secrets.TEST_RESULTS_WALLET }}
+        run: |
+          echo "$TEST_RESULTS_WALLET" | base64 -d > wallet.json
+        working-directory: neofs-testcases
```

Comment:
Why this dir?

Reply:
At the moment ./tools/src/process-allure-reports.py is a part of neofs-testcases. That's the reason why I have to checkout neofs-testcases and use this script to upload test-result to NeoFS